mod.ts

```diff
@@ -2,7 +2,7 @@ import type {
   Context,
   Middleware,
   Transformer,
-} from "https://deno.land/x/grammy@v1.21.2/mod.ts";
+} from "https://deno.land/x/grammy@v1.30.0/mod.ts";
```

Comment:
**suggestion (blocking):** let's go with `https://lib.deno.dev/x/grammy@v1/mod.ts` instead so we don't have to do this again in the future

Reply:
That works too, but it will trigger an LSP suggestion:
```ansi
The import of "https://lib.deno.dev/x/grammy@v1/mod.ts" was redirected to "https://deno.land/x/grammy@v1.30.0/mod.ts". deno(redirect) [Ln 5, Col 8]
```
Perhaps we could use `npm:grammy@1` instead?

Reply:
Okay